infra/main.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": { 
+            "value": "${AZURE_LOCATION}"
+        },
+        "environmentName": {
+          "value": "${AZURE_ENV_NAME}"
+        },
+        "principalId":{
+            "value": "${AZURE_PRINCIPAL_ID}"
+        },  
+        "repositoryUrl": {
+            "value": "https://github.com/<your-account>/azure-sql-db-session-recommender"

```

Comment:
All of these values should be driven from env vars, not ask the user to replace values in this file

Reply:
Modified.